package.json

```diff
@@ -10,7 +10,7 @@
     "build": "webpack --env=prod",
     "test": "npm-run-all --parallel --continue-on-error test-app test-api",
     "test-app": "karma start karma.conf.js",
-    "test-api": "./node_modules/.bin/mocha --recursive test/api",
+    "test-api": "./node_modules/.bin/mocha test/api --recursive --require test/api/api-test-globals",
```

Comment:
I vote leave it this way. I know this difference matters with `eslint` so there is some difference. Maybe it doesn't affect `mocha` but better safe than sorry.